packages/blocks/tailwind.config.ts

```diff
@@ -2,7 +2,7 @@ import type { Config } from 'tailwindcss';
 import { theme } from '@viamrobotics/prime-core/theme';
 
 export default {
-  content: ['./src/**/*.{ts,svelte}'],
+  content: ['./src/**/*.{ts,svelte}', '../core/dist/**/*.{ts,js,svelte}'],
   theme,
```

Comment:
Why not add the core `node_modules` here?

Reply:
Is there something from `node_modules` we should be watching?